pkg/testing/tools/tools.go

```diff
@@ -121,11 +126,15 @@ func InstallAgentForPolicy(t *testing.T, installOpts atesting.InstallOpts, agent
 	}
 	t.Logf(">>> Ran Enroll. Output: %s", output)
 
+	timeout := 5 * time.Minute
+	if deadline, ok := ctx.Deadline(); ok {
+		timeout = deadline.Sub(time.Now().Add(-500 * time.Millisecond))
+	}
```

Comment:
I like this idea, allowing the context to control the timeout. But why the adjustment?

Reply:
i removed it